src/clojure_tictactoe/tictactoe.clj

```diff
@@ -1,31 +1,78 @@
 (ns clojure-tictactoe.tictactoe
-  (:gen-class))
+  (:gen-class)
+  (:require [clojure.string :as string]))
 
 
-(def welcome-message 
+(def welcome-message
   "Welcome to the game of TicTacToe! \n")
 
-(def instructions-message 
+(def instructions-message
  "Instructions: 2 players take turns placing markers, try to get three of your markers in a row to win! \n
 
-Examples: 
+Examples:
 
   Vertical      Horizontal      Diagonal
 
-   | x |     |    |   |     |  x |   |     
-===+===+===  | ===+===+===  | ===+===+===  
-   | x |     |  x | x | x   |    | x |     
-===+===+===  | ===+===+===  | ===+===+===  
+   | x |     |    |   |     |  x |   |
+===+===+===  | ===+===+===  | ===+===+===
+   | x |     |  x | x | x   |    | x |
+===+===+===  | ===+===+===  | ===+===+===
    | x |     |    |   |     |    |   | x  \n\n ")
 
 
 (defn print-game-intro []
   (println welcome-message)
   (println instructions-message))
 
+;TODO make this DRYer when you have minute and a better understanding of Clojure
```

Comment:
I'd leave these to-do comments out when you make your PRs. The idea of a PR is that you're requesting for your code to be pulled into master, so you usually try to eliminate things like this that you wouldn't want to see in the master branch of the code base. Right now you're working on it by yourself so the distinction may not be that big, but basically imagine that you are sharing this code base with a team of other devs.

Reply:
Got it! Thanks!